manifests/init.pp

```diff
@@ -76,6 +76,8 @@
     info('puppet_agent performs no actions if a package_version is not specified on Puppet 4')
   } elsif $package_version == undef and $is_pe {
     info("puppet_agent performs no actions if the master's agent version cannot be determed on PE 3.x")
+  } elsif $facts['pe_server_version'] != undef {
+    info('puppet_agent performs no actions on PE infrastructure nodes to prevent a mismatch between agent and PE components')
```

Comment:
@MikaelSmith thoughts on why these are logging at info?  I was thinking of upgrading them to [warning](https://docs.puppet.com/puppet/3.8/function.html#warning).

Reply:
I forget if that's something we got complaints about. The 1st one was info so that you wouldn't get warnings immediately after upgrading from Puppet 3. The 2nd should probably be a warning.

Reply:
I don't think it should be a warning. We should only use warnings for things that a user is doing incorrectly that we believe are actionable. In this case, it's a totally reasonable behavior to classify "all nodes" with this module, and the module knows how to handle that case correctly (now). Warning here and forcing users to create a new "everything-but-pe-infrastructure" group in order to silence that warning is just annoying busywork.

Reply:
I think that makes sense the message I'm adding isn't actionable it's informational.